lib/matplotlib/texmanager.py

```diff
@@ -60,12 +60,6 @@
 
 DEBUG = False
 
-if sys.platform.startswith('win'):
```

Comment:
I thought that `cmd_split` wasn't used anywhere. But I'm probably wrong in trying to do too many things in this pull request. Maybe it's possible that users are depending on it?